docs/src/dataDisplay/headerDeprecated/headerDeprecated.jsx

```diff
@@ -6,10 +6,10 @@ import {
 } from 'react-cm-ui';
```

Comment:
The component is still `<Header>` not `<HeaderDeprecated>` ... are there near future plans to deprecate it?

Reply:
Yeah, it is a soft deprecation; bwahaha!
We should only be using the Typography component now.